### ISSUE TITLE
Checkout block checkboxes and radio buttons cannot be focused via keyboard in safari

### ISSUE THREAD

**Issue by senadir** [woothemes/woocommerce#57547]
## **Describe the bug**

In safari, you can't focus on checkboxes or radio buttons using keyboard and tabbing, this would only navigate you in fields and buttons, but not checkboxes.

## **To Reproduce**

1. In safari, enable account creation during checkout.
2. Incoginito, go to checkout, select email field, try tabbing into create account checkbox, it won't work.
3. Continue to "use shipping for billing" it won't work
4. Continue tabbing, it will skip shipping and payment methods.

## **Expected behavior**

Those fields should be tabbable.

## Environment

Please fill out any of the below that applies

**Operating System/Device**

- [ ] Windows
- [X] MacOS
- [ ] Linux
- [ ] iPhone
- [ ] Samsung
- [ ] Other (add in additional context field)

**Version:** 

**Browser**

- [ ] Desktop Edge
- [ ] Desktop Chrome/Chromium
- [X] Desktop Safari
- [ ] Mobile Edge
- [ ] Mobile Chrome/Chromium
- [ ] Mobile Safari
- [ ] Other (add in additional context field)

**Version:** 

## WordPress Environment

<details><summary>System Status report</summary>

</details>

**Comment by opr** [comment]
Looks like this is purposeful Safari behaviour: [https://github.com/shoelace-style/shoelace/issues/840#issuecomment-1199194223](https://github.com/shoelace-style/shoelace/issues/840#issuecomment-1199194223)

Updating settings as per that thread allows them to be tabbed.
